src_features/signMessageEIP712/commands_712.c

```diff
@@ -14,6 +14,24 @@
 #include "filtering.h"
 #include "common_712.h"
 #include "common_ui.h"  // ui_idle
+#include "manage_asset_info.h"
+
+// APDUs P1
+#define P1_COMPLETE 0x00
+#define P1_PARTIAL  0xFF
+
+// APDUs P2
+#define P2_DEF_NAME               0x00
+#define P2_DEF_FIELD              0xFF
+#define P2_IMPL_NAME              P2_DEF_NAME
```

Comment:
Because these two values end up being the same (as defined in the APDU specs) but they didn't necessarily have to be, and having two defines makes the code more readable where they end up being used in code.